.docker/web/Dockerfile

```diff
@@ -8,9 +19,10 @@ RUN apt-get update -y && \
 RUN apt-get update -y && \
     apt-get install -y golang
 
-COPY .docker/web/vhost.conf /etc/apache2/sites-available/000-default.conf
-
-WORKDIR /srv/app
-
-RUN mkdir -p /srv/app/storage/code
-RUN chmod -R ug+rwx /srv/app/storage/code && chown -R www-data:www-data /srv/app && a2enmod rewrite
+# Packages for formatting code using Prettier (JS, CSS, etc)
+RUN curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | \
+    tee /etc/apt/sources.list.d/yarn.list && \
+    apt-get update -y && \
+    curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
+    apt-get install -y nodejs yarn && yarn install
```

Comment:
I think it started with better performance when starting scripts directly from yarn instead of npm. But that was still too slow so I started calling prettier directly from node_modules, but the docker commands stayed. 

Will change it to npm and do some quick tests.

Reply:
Switched in 4a28851